features/scrubber.feature

```diff
@@ -1,41 +1,41 @@
 Feature: Helloworld
```

Comment:
Should the feature be called `Helloworld`?

Reply:
Good catch!